### ISSUE TITLE
Review "Document contain at most one banner landmark" axe rule

### ISSUE THREAD

**Issue by bjankord** [cerner/terra-toolkit#243]
# Issue Description
Teams using Terra UI components are running into accessibility failures related to the following rule:
```
Document contain at most one banner landmark
```
This is caused by multiple `<header />` elements on page. We need to determine what action we want to take to resolve this.

## Additional Context / Screenshots
Here is a sample of the error ouput:

```js
AssertionError: expected no accessibility violations but got:
{
  "help": "Document contain at most one banner landmark",
  "nodes": [
    {
      "all": [],
      "failureSummary": "Fix any of the following:\n  Document has more than one banner landmark",
      "impact": "moderate",
      "html": "<html lang=\"de\" dir=\"ltr\">",
      "none": [],
      "any": [
        {
          "id": "page-no-duplicate-banner",
          "data": null,
          "message": "Document has more than one banner landmark",
          "impact": "moderate",
          "relatedNodes": [
            {
              "html": "<header class=\"ActionHeaderContainer-module__flex-header___QeMsy\"><div class=\"ActionHeaderContainer-module__flex-fill___1jBA6\"><div class=\"ActionHeaderContainer-module__title-container___3eBPC\"><h1 class=\"ActionHeaderContainer-module__title___1t3UH\">Erfolg.</h1></div></div></header>",
              "target": [
                ".ActionHeaderContainer-module__flex-header___QeMsy"
              ]
            },
            {
              "html": "<header class=\"Header__flex-header___31sRw\" data-terra-clincial-action-header=\"true\">",
              "target": [
                ".Header__flex-header___31sRw"
              ]
            }
          ]
        }
      ],
      "target": [
        "html"
      ]
    }
  ],
  "impact": "moderate",
  "description": "Ensures the document has no more than one banner landmark",
  "helpUrl": "https://dequeuniversity.com/rules/axe/3.0/landmark-no-duplicate-banner?application=axeAPI",
  "id": "landmark-no-duplicate-banner",
  "tags": [
    "cat.semantics",
    "best-practice"
  ]
}
```

In this case, the app has 2 Terra components that render a `<header />` element.

**Comment by bjankord** [comment]
My thoughts are this rule seems to be a bit over-opinionated, but I'm curious what others think. If we want to resolve this without disabling the rule, we just need to switch from rendering a `<header />` and render a `<div />` instead.

**Comment by emilyrohrbough** [comment]
When reviewing the axe-core rule for the `page-no-duplicate-banner`, it appears [this is tagged](https://github.com/dequelabs/axe-core/blob/05e4f8254c53f3f97d1432b838c2a94edee8ab66/lib/rules/landmark-no-duplicate-banner.json#L4) as `best-practices` and `cat.semantics`, but when comparing to our guidelines specifies accessibility for `WCAG 2.0`, `Section 508` and `Color Contrast Checker`. 

Given that, I think disabling this rule is fine, however, the would probably be better to scope the accessibility checks to these guidelines: https://github.com/dequelabs/axe-core/blob/05e4f8254c53f3f97d1432b838c2a94edee8ab66/doc/API.md#parameters-1

Tag options: `wcag2a`, `wcag2aa`, `section508`, `best-practice`

**Comment by ryanthemanuel** [comment]
+1 @emilyrohrbough.  We should reevaluate this configuration and make sure what we're setting up is what we're trying to align with.
